src/pages/[marketplace]/deals/index.tsx

```diff
@@ -74,17 +79,17 @@ const Deals: NextPageWithLayout = () => {
 
 		const repaidRatio = new Ratio(principal.toNumber(), principalAmountRepaid.toNumber());
 
-		return repaidRatio.apply(new Big(100));
+		return repaidRatio.apply(100);
 	};
 
 	const mapDeal = useCallback(
 		({ address, name, principal, goLiveAt, principalAmountRepaid }: Deal) => {
 			return {
 				key: address.toString(),
 				name: name,
-				amount: toUIAmount(principal).toNumber(),
-				date: formatTimestamp(goLiveAt, locales as string[]),
-				paid: dealRepaidRatio(principal, principalAmountRepaid),
+				amount: toUIAmount(new Big(principal)).toNumber(),
```

Comment:
you don't have to change but just a tip: you can do Big(principal). the new is not necessary